metadata-ws/src/main/resources/schema.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS security_user (
+  preferred_username VARCHAR(255) NOT NULL,
+  role  VARCHAR(255),
+  PRIMARY KEY (preferred_username)
+);
```

Comment:
I may be missing something here, but why is this schema necessary? Since `SecurityUser` is an `@Entity`, shouldn't the necessary tables just be created for it?